src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryHolder.java

```diff
@@ -29,16 +29,19 @@
 import java.util.Set;
 import javax.management.openmbean.TabularData;
 
+import org.apache.cassandra.io.util.FileUtils;
 import org.apache.cassandra.tools.NodeProbe;
 
 public class CompactionHistoryHolder implements StatsHolder
 {
     public final NodeProbe probe;
     public List<String> indexNames;
+    public final boolean humanReadable;
```

Comment:
this does not need to be public. I get that you probably copy how it is elsewhere but for other cases it has to be public, for this it does not.